about.html

```diff
@@ -7,67 +7,123 @@
   <title>Sprint Challenge - About</title>
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+  <link rel="stylesheet" href="css/about.css">
 
 </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
 
 
-    <img src="img/about-plan.png" alt="strategy">
+<body>
+        <div class="All">
+        <div class="container about-page">
+            <header>
+                <div class='Head'>
     
-    Strategy
+                <div class='MHDiv'>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
+                <nav class='Nav'>
+                            <a href=""><img src="img/lambda-black.png"></a>
```

Comment:
Your HTML spacing is a little bit off. Consider using VS code extensions such as `Beautify` or `Prettier`. They'll automatically fix the spacing for you.